packages/stripe/src/Adaptor.js

```diff
@@ -0,0 +1,331 @@
+import { expandReferences } from '@openfn/language-common/util';
+import * as util from './Utils';
+
+/**
+ * State object
+ * @typedef {Object} HttpState
+ * @property data - the parsed response body
+ * @property response - the response from the HTTP server, including headers, statusCode, body, etc
+ * @property references - an array of all previous data objects used in the Job
+ **/
+
+/**
+ * List items from Stripe
+ * @example <caption>List all payment intents</caption>
+ * list('/payment_intents');
```

Comment:
Isn't this a resource, rather than a path? Are there examples where a path is needed?

I think you should rename the parameter and remove the `/` from the examples